src/deepsparse/transformers/README.md

```diff
@@ -139,31 +139,36 @@ response.text
 >> '{"score":0.9534820914268494,"start":8,"end":14,"answer":"batman"}'
 ```
 
-### Text Generation
-The text generation task generates a sequence of tokens given the prompt. Popular text generation LLMs (Large Language Models) are used
+### Text Generation 
+The text generation task generates a sequence of tokens given the prompt. Popular text generation Large Language Models (LLMs) are used
 for the chatbots (the instruction models), code generation, text summarization, or filling out the missing text. The following example uses a sparsified text classification
-OPT model to complete the prompt
+OPT model to complete the prompt.
 
-[List of available SparseZoo Text Generation Models](
-https://sparsezoo.neuralmagic.com/?useCase=text_generation)
+#### KV Cache Injection
+Please note, that to take the full advantage of the speedups provided by the DeepSparse Engine, it is essential to run inference using a model with the KV cache support. 
+If you are using one of the pre-sparsified models from SparseZoo ([list of available SparseZoo Text Generation Models](
+https://sparsezoo.neuralmagic.com/?useCase=text_generation)), you will automatically benefit from the KV cache support speedups.
+However, if you are sparsifying your custom model, you may want to add the KV cache support to your model. This will be extremely beneficial when it comes to the inference speed.
```

Comment:
```suggestion
However, if you are sparsifying your custom neural network, you may want to add the KV cache support to your model. This will be extremely beneficial when it comes to inference speed.
```